Vizzuality/trase
# Feat/country profiles

## Description

Included some new/redefined widgets for country profiles, included a checkbox as there are multiple tasks related to this. 

- [x] Include summary widget with new country profile data
https://www.pivotaltracker.com/story/show/171568760

- [x] Map widget 
https://www.pivotaltracker.com/story/show/171568576

- [x] Table widget 
https://www.pivotaltracker.com/story/show/171568384

- [x] New line widget (based on previous widgets)
https://www.pivotaltracker.com/story/show/171568521

- [x] Data for each of the widgets
https://www.pivotaltracker.com/story/show/171568904

## Testing instructions

Test both source and importer country profiles and make sure widgets appear. And also switch context and year and profile pages should update with new data.
